docs/src/content/docs/protocol/deployments.mdx

```diff
@@ -2,143 +2,6 @@
-| `cometbls-light-client` | [`osmo13y8ke43r46cnnc86gy20sg5ucdpxux9jc3nazyk447jehrz4rtqqk8xckr`](https://celatone.osmosis.zone/osmo-test-5/contracts/osmo13y8ke43r46cnnc86gy20sg5ucdpxux9jc3nazyk447jehrz4rtqqk8xckr) |
-| `tendermint-light-client` | [`osmo1pru4puw0uw54cqjahld674a60z9un29k9jfqh36804lryn2m9m9s0femjr`](https://celatone.osmosis.zone/osmo-test-5/contracts/osmo1pru4puw0uw54cqjahld674a60z9un29k9jfqh36804lryn2m9m9s0femjr) |
-| `state-lens-ics23-mpt-client` | [`osmo1mgfar57xxq2xgy6yrm807njxuwr90g8sx67jal0ar5lqpst8597q47e43d`](https://celatone.osmosis.zone/osmo-test-5/contracts/osmo1mgfar57xxq2xgy6yrm807njxuwr90g8sx67jal0ar5lqpst8597q47e43d) |
-| `ucs03-zkgm` | [`osmo1yx9yfftvh98khe4h9z3azyn2wgveaulhh845aq7z26uvzaqm286s70azd8`](https://celatone.osmosis.zone/osmo-test-5/contracts/osmo1yx9yfftvh98khe4h9z3azyn2wgveaulhh845aq7z26uvzaqm286s70azd8) |
-
-### Neutron Testnet (`pion-1`)
-| Name | Address |
-| --- | --- |
-| `ibc-union` | [`neutron1ffvngvnsvmn7n0aqe48t6xxfy98m7s85tlxafektdt3cv7taqxuqu6d3rt`](https://neutron.celat.one/pion-1/contracts/neutron1ffvngvnsvmn7n0aqe48t6xxfy98m7s85tlxafektdt3cv7taqxuqu6d3rt) |
-| `cometbls-light-client` | [`neutron1yym4ngvwyv9u7es8lc474awx3wp5ty6tnevjxt0ut8m8ke9r52ysrqde0h`](https://neutron.celat.one/pion-1/contracts/neutron1yym4ngvwyv9u7es8lc474awx3wp5ty6tnevjxt0ut8m8ke9r52ysrqde0h) |
-| `ucs03-zkgm` | [`neutron1cwc570jxrdmvhcewl6puzlfdc62s2gzky8z03xsex80ew77ne88sm4qa43`](https://neutron.celat.one/pion-1/contracts/neutron1cwc570jxrdmvhcewl6puzlfdc62s2gzky8z03xsex80ew77ne88sm4qa43) |
-
-### Channels
-
-| Protocol  | From             | To              | Channel  |
-|---------- |----------------- |----------       |--------- |
-| ucs03     | 17000            | 11155111        | 9        |
-| ucs03     | 11155111         | 17000           | 4        |
-| ucs03     | 17000            | union-testnet-9 | 8        |
-| ucs03     | union-testnet-9  | 17000           | 7        |
-| ucs03     | 17000            | elgafar-1       | 11       |
-| ucs03     | elgafar-1        | 17000           | 15       |
-| ucs03     | 17000            | bbn-test-5      | 26       |
-| ucs03     | bbn-test-5       | 17000           | 2        |
-| ucs03     | osmo-test-5      | union-testnet-9 | 1        |
-| ucs03     | union-testnet-9  | osmo-test-5     | 13       |
-| ucs03     | 17000            | osmo-test-5     | 27       |
-| ucs03     | osmo-test-5      | 17000           | 2        |
-| ucs03     | elgafar-1        | osmo-test-5     | 17       |
-| ucs03     | osmo-test-5      | elgafar-1       | 3        |
-| ucs03     | elgafar-1        | union-testnet-9 | 18       |
-| ucs03     | union-testnet-9  | elgafar-1       | 14       |
-| ucs03     | elgafar-1        | bbn-test-5      | 25       |
-| ucs03     | bbn-test-5       | elgafar-1       | 9        |
+:::warning
 
+The contents of this file have been moved to [deployments.json](https://github.com/unionlabs/union/blob/main/deployments.json).
```

Comment:
Let's postpone putting this message here. For now, perhaps just hide the page?
https://github.com/unionlabs/union/issues/3725